python/nano/src/bigdl/nano/automl/pytorch/objective.py

```diff
@@ -183,19 +147,57 @@ def _fix_data_args(self,
                               "or `val_dataloaders` to `trainer.search(datamodule=...)`")
         return train_dataloaders, val_dataloaders, datamodule
 
+    def _auto_optimize(self, model, scores):
+        Score = namedtuple("Score", self.target_metric)
+        best_score = Score(*scores)
+        model.eval()
+
+        # here we suppose nn.model is attached to plmodel by attribute "model"
+        original_model = model.model
+        for optimization in ['openvino', 'onnxruntime', 'jit']:
+            # may enable more optimizations later
+            try:
+                if optimization == 'jit':
+                    optim_model = Trainer.trace(original_model, input_sample=self.input_sample,
+                                                accelerator=optimization,
+                                                use_ipex=True, channels_last=False)
```

Comment:
I am not quite sure if states `channels_last=False` by default is correct.

Reply:
Do you think here use a try-catch to capture "channels_last=True" error is better or export channels_last parameter to user is better?